src/uwtools/rocoto.py

```diff
@@ -374,7 +375,7 @@ def _insert_doctype(self, xml: str) -> str:
             lines.insert(1, doctype)
         return "\n".join(lines)
 
-    def _set_and_render_jobname(self, config: dict, taskname: str) -> None:
+    def _set_and_render_jobname(self, config: dict, taskname: str) -> dict:
```

Comment:
Needs a `:return:` entry in the docstring, or the first docstring line updated to state `Return ...`.